packages/dds/signal/src/sharedSignalFactory.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type {
+	IChannelAttributes,
+	IFluidDataStoreRuntime,
+	IChannelFactory,
+	IChannelServices,
+} from "@fluidframework/datastore-definitions/internal";
+
+import type { ISharedSignal } from "./interfaces.js";
+import { SharedSignal } from "./sharedSignal.js";
+
+/**
+ * @internal
+ */
+export const pkgName = "@fluidframework/signal";
+const pkgVersion = "0.25.0";
+
```

Comment:
can we delete these. they don't look used, and there is an automatically generated one in packages/dds/signal/src/packageVersion.ts